packages/blocks-ui/src/side-panel.js

```diff
@@ -154,3 +121,21 @@ export default ({
     </Tabs>
   </section>
 )
+
+// Base sytles for tabs
+const baseTabStyles = {
```

Comment:
I considered making a `getBaseTabStyles` function that takes the `activeTab` and tab index as arguments, but it felt a little bit _too clever_ and less readable.

This is not perfect, but you can easily see what the `baseStyles` are composed with (border styles) for each tab - which is probably fine here.